tests/utils.py

```
@@ -714,6 +714,7 @@ def create_windows2022_vm_using_existing_dv(
     vm_name: str,
     cpu_model: str | None = None,
     existing_data_volume: DataVolume | None = None,
+    check_running_vm: bool = True,
```

Ruff (0.16.0)

[warning] Boolean-typed positional argument in function definition

(FBT001)